Examples/checkers.lisp

```diff
@@ -0,0 +1,446 @@
+;;;;  Copyright (c) 2020 Daniel Kochmański
+;;;;
+;;;;    License: BSD-2-Clause.
+
+(defpackage #:clim-demo.checkers
+  (:use #:clim-lisp #:clim)
+  (:export #:run-checkers #:clim-checkers)
+  (:import-from #:alexandria
+                #:if-let #:when-let #:when-let* #:iota))
+(in-package #:clim-demo.checkers)
+
+(defparameter +square-side+ 72)
+
+(defparameter +cols+ 8)
+(defparameter +rows+ 8)
+
+(defparameter +col-labels+
+  (mapcar (lambda (n)
+            (format nil "~a" n))
+          (alexandria:iota +cols+ :start 1)))
```

Comment:
`iota` is imported, no need for the package prefix.

Reply:
fixed